projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.html

```diff
@@ -19,52 +21,56 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-4">{{ 'Query details' | rpxTran
+    <form [formGroup]="responseFormGroup">
       <div class="govuk-form-group response-textarea"
-           [class.govuk-form-group--error]="responseFormGroup.get('response')?.errors?.length">
+           [class.govuk-form-group--error]="submitted && responseFormGroup.get('response')?.errors">
         <div class="govuk-label-wrapper">
           <label class="govuk-label govuk-!-font-weight-bold" for="query-response">
             {{ 'Response message' | rpxTranslate }}
           </label>
         </div>
         <p id="query-response-error" class="govuk-error-message"
-           *ngIf="responseFormGroup.get('response')?.errors?.length">
-          <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'Enter a response message' | rpxTranslate }}
+           *ngIf="submitted && responseFormGroup.get('response')?.hasError('required')">
+          <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }} </span>{{ 'Enter a response message' | rpxTranslate }}
         </p>
-        <textarea [formControlName]="'response'"
-          class="govuk-textarea" [class.govuk-textarea--error]="responseFormGroup.get('response')?.errors?.length"
+        <textarea [formControlName]="'response'" class="govuk-textarea"
+          [class.govuk-textarea--error]="submitted && responseFormGroup.get('response')?.hasError('required')"
           id="query-response" name="query-response"
           rows="5" aria-describedby="query-response-hint query-response-error">
         </textarea>
       </div>
 
       <div>
-        <div class="govuk-form-group" [class.govuk-form-group--error]="responseFormGroup.get('documents')?.errors?.length">
+        <div class="govuk-form-group"
+             [class.govuk-form-group--error]="submitted && responseFormGroup.get('documents')?.errors">
           <div class="govuk-label-wrapper">
             <label class="govuk-label" for="add-documents">
               <div class=" govuk-!-font-weight-bold">{{ 'Add document (optional)' | rpxTranslate }}</div>
               <div>{{ 'Attach a document to this message' | rpxTranslate }}</div>
             </label>
           </div>
-          <p id="add-documents-error" class="govuk-error-message" *ngIf="responseFormGroup.get('response')?.errors?.length">
-            <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'File error message' | rpxTranslate }}
+          <p *ngIf="submitted && responseFormGroup.get('documents')?.errors"
+             id="add-documents-error" class="govuk-error-message"
+          >
+            <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }} </span>{{ 'File error message' | rpxTranslate }}
           </p>
 
           <input class="govuk-file-upload" id="add-documents" name="add-documents" type="file">
         </div>
       </div>
-    </div>
+    </form>
+    <button class="govuk-button" data-module="govuk-button">
+      {{ 'Next' | rpxTranslate}}
```

Comment:
@MateiHMCTS - minor point on spacing of `rpxTranslate` and `}}`; same on Line 69